documentation/src/main/asciidoc/topics/proc_cli_registering_protobuf_schema.adoc

```diff
@@ -22,8 +22,12 @@ include::protobuf/person.proto[]
 +
 [source,options="nowrap",subs=attributes+]
 ----
-[//containers/default]> cd caches/___protobuf_metadata
-[//containers/default/caches/___protobuf_metadata]> ls
+[//containers/default]> ls 
+caches
+counters
+configurations
+schemas
+tasks
+[//containers/default]> ls 
```

Comment:
Thanks for the contribution @Scuilion, but I'm not sure if this change is correct. I believe the intent of the original example is to change to the `caches/___protobuf_metadata]` context and then display all registered `*.proto` files. Your addition simply shows the result of executing `ls` in the root context, which isn't necessary to verify the presence of `person.proto`.

\\cc @oraNod

Reply:
Indeed, thanks for the contribution @Scuilion! Ryan is correct in that the current change you're suggesting doesn't actually verify `person.proto`. That said the verification step does need to change because it was written before the `schemas` directory existed. `___protobuf_metadata` is still there, but is hidden.

We could change this to `cd schemas` then `ls` which shows all the Protobuf schema files.